cli/cmd/root.go

```diff
@@ -109,3 +108,28 @@ func (s *options) appendIfNotEmpty(newOption string) {
 		*s = append(*s, newOption)
 	}
 }
+
+func runDailyVersionCheck() {
+	vChecker := version.NewVersionChecker()
+	cliMsgPrinted, cliChecked := vChecker.CheckCLIVersion(Version, true)
+
+	keptnVersion, err := getInstalledKeptnVersion()
+	if err != nil {
+		logging.PrintLog(err.Error(), logging.InfoLevel)
+		return
+	}
```

Comment:
Same as with my comment down below: We don't necessarily need to `return` here, just keep in mind that we can not check for a new version if keptnVersion is nil or something like that ;)